setup.py

```diff
@@ -46,19 +46,19 @@
     include_package_data=True,
     zip_safe=False,
     license='MPL-2.0',
-    python_requires=">=3.6",
+    python_requires=">=3.7",
     classifiers=[
         'Development Status :: 5 - Production/Stable',
         'Intended Audience :: Developers',
         'License :: OSI Approved :: Mozilla Public License 2.0 (MPL 2.0)',
         'Natural Language :: English',
         'Programming Language :: Python',
         'Programming Language :: Python :: 3',
```

Comment:
```suggestion
        'Programming Language :: Python :: 3',
        'Programming Language :: Python :: 3 :: Only',
```

A

Reply:
sure but I'm not sure anyone cares about that classifier any more

Reply:
Fair enough, I'm never really sure with classifiers.

A